### ISSUE TITLE
Outline and regions

### ISSUE THREAD

**Issue by jonathanv81** [mgr32/PsISEProjectExplorer#81]
Great extension! I really like to have all my scripts available in one click and be able to see the functions in them.
My issue is more a request or a wish: I would like to have an outline of the script I'm working on. Instead of the functions in alphabetic order, to have the list of the regions and functions in the order in which they appear in the script hierarchically (like you already have for functions and sub-functions).
Would that be possible? Does it require a lot of work? If you can give me some pointers, I will look into it although I'm not a developer.
Thank you for your great work!

**Comment by mgr32** [comment]
Hi @jonathanv81, thanks for the request. If I understand correctly, there should be an additional level of hierarchy for regions and order of function names should be preserved. Sounds like a good idea and should not be a lot of work. Will look at it when time allows.

**Comment by jonathanv81** [comment]
Yes, just like how you display a function with sub-functions as a tree, it would be regions with functions inside that region showing at a lower level in the tree. Alphabetical or chronological order could be an option so as to cater to every taste :)
Thank you for your time and support!

**Comment by mgr32** [comment]
Done the following in 1.5.0:
* Added a button that controls whether Powershell items (like functions or classes) should be sorted by name (old behaviour) or by line number (new behaviour, default)
* If Powershell items are sorted by line number, regions are also displayed in tree hierarchy (nested regions are also supported).
